src/main/java/net/masterthought/cucumber/TagObject.java

```diff
@@ -90,45 +90,49 @@ public String getDurationOfSteps() {
     public int getNumberOfSteps() {
         int totalSteps = 0;
         for (ScenarioTag scenario : scenarios) {
-            if (scenario.hasSteps()) {
+            if (!scenario.getScenario().isBackground() && scenario.hasSteps()) {
                 totalSteps += scenario.getScenario().getSteps().size();
             }
         }
         return totalSteps;
     }
 
-    public int getNumberOfStatus(Status status) {
+    private int getNumberOfStatusExcludingBackGround(Status status) {
+        return Util.findStatusCount(getStatusesByExcludingBackGround(), status);
+    }
+
+    public int getNumberOfStatusIncludingBackGround(Status status) {
         return Util.findStatusCount(getStatuses(), status);
     }
 
     /** No-parameters method required for velocity template. */
     public int getNumberOfPasses() {
-        return getNumberOfStatus(Status.PASSED);
+        return getNumberOfStatusExcludingBackGround(Status.PASSED);
```

Comment:
I'm not sure about returning value without backgorunds - by default I would expect to have all items but I will have to check with code to be sure about consequences...